scripts/scil_compute_local_tracking.py

```diff
@@ -13,6 +13,23 @@
 Algo 'prob': a direction drawn from the empirical distribution function defined
 from the SF.
 
+The local tracking algorithm can also run on the GPU using the --use_gpu
+option (experimental). By default, GPU tracking behaves the same as
+DIPY. Below is a list of known divergences between the CPU and GPU
+implementations:
+    * Backend: The CPU implementation uses DIPY's LocalTracking and the
+        GPU implementation uses an in-house OpenCL implementation.
+    * Algo: For the GPU implementation, the only available algorithm is
+        Algo 'prob'.
+    * Tracking sphere: The only sphere available for GPU tracking is
+        `symmetric724` and `--sub_sphere` is not available for GPU tracking.
```

Comment:
Yes it is pretty easy. Didn't do it because I wanted to make a fast first pass, but then I hit a big wall of problems, so... anyway. The idea was just to merge both together. Supporting other spheres can be done at some other time I think (could be on my scilpy hackathon stack).

Reply:
It's all good with me, I can add it to the list. And I can also do the addition, we'll see on thursday .